runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsConfig.java

```diff
@@ -28,20 +28,78 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("inventorytags")
 public interface InventoryTagsConfig extends Config
 {
-	enum DisplayMode
+	String GROUP = "inventorytags";
+
+	int OPACITY_LIMIT_MIN = 0;
+	int OPACITY_LIMIT_MAX = 255;
```

Comment:
I suppose these can be replaced with `ColorUtil.(MIN|MAX)_RGB_VALUE` given opacity is also represented by the same range